pinot-query-runtime/src/main/java/org/apache/pinot/query/service/server/QueryServer.java

```diff
@@ -125,9 +132,9 @@ public void shutdown() {
 
   @Override
   public void submit(Worker.QueryRequest request, StreamObserver<Worker.QueryResponse> responseObserver) {
-    Map<String, String> requestMetadata;
+    Map<String, String> reqMetadata;
```

Comment:
Why was the name of variable changed ?

Reply:
It was deleted by mistake. I had the same issue in https://github.com/apache/pinot/pull/15245. I had to be doing something strange with git. I'm going to take a second look

Reply:
Just to be able to write the try bellow in a single line

```
    try (QueryThreadContext.CloseableContext queryTlClosable = QueryThreadContext.openFromRequestMetadata(reqMetadata);

```